src/ResourceManager/HDInsight/Commands.HDInsight/JobCommands/GetAzureHDInsightJobOutputCommand.cs

```diff
@@ -133,25 +134,38 @@ private static string Convert(Stream stream)
 
         internal IStorageAccess GetDefaultStorageAccess(string resourceGroupName, string clusterName)
         {
+            var StorageAccountSuffix = "";
```

Comment:
@epkalyanr, please make the suggested change. This information is provided to us as part of the environment. We don't have this context in the SDK, so we don't have any other option. Since we do have the context in PS, we should use that instead!